<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>7</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>228</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tworzenie i macierz sąsiedztwa</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ilość wierzchołków:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="vertsNumberSpin">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createNewGraphButton">
            <property name="text">
             <string>utwórz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>M=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="MatrixView" name="matrixView" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <attribute name="label">
        <string>Macierz incydencji</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>280</width>
             <height>208</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="ConnectionMatrixView" name="connectionMatrix" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>300</width>
         <height>228</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Lista sąsiedztwa</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>LIsta sąsiedztwa:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>280</width>
             <height>169</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="ListView" name="neLIST" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="AddConnection" name="addConnectionView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <attribute name="label">
        <string>Generator</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Ilość wierzchołków:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="vertexNumberInGenerator">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="generate">
            <property name="text">
             <string>generuj</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <attribute name="label">
        <string>Silnie spójne składowe i cykle</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>składowe:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>cykle:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QScrollArea" name="scrollArea_3">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>212</width>
               <height>99</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="strongComponents">
                <property name="text">
                 <string/>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QScrollArea" name="scrollArea_4">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>212</width>
               <height>99</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="cycles">
                <property name="text">
                 <string/>
                </property>
                <property name="textFormat">
                 <enum>Qt::RichText</enum>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <attribute name="label">
        <string>BF i generator</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Graf o losowych wagach [-5,20] na podstawie największej spójnej składowej:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generateFromGraphToBF">
          <property name="text">
           <string>generuj</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Podaj wierzchołek:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="BF_vertex_ID"/>
          </item>
          <item>
           <widget class="QPushButton" name="checkDistance">
            <property name="text">
             <string>wyznacz</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_5">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>280</width>
             <height>101</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="distanceLabelBF">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <attribute name="label">
        <string>Johnson &amp;&amp; Floyd-Warshall</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="johnsonApply">
          <property name="text">
           <string>Johnson</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="floydWarshall">
          <property name="text">
           <string>Floyd-Warshall</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>D=</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea_6">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_6">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>256</width>
               <height>140</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="DistanceMatrix" name="distanceMatrix" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_8">
       <attribute name="label">
        <string>Sortowanie topologiczne</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="sortTrigget">
          <property name="text">
           <string>sortuj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sortResult">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="GraphPainter" name="graphView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MatrixView</class>
   <extends>QWidget</extends>
   <header>matrixview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListView</class>
   <extends>QWidget</extends>
   <header>listview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AddConnection</class>
   <extends>QWidget</extends>
   <header>addconnection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConnectionMatrixView</class>
   <extends>QWidget</extends>
   <header>connectionmatrixview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphPainter</class>
   <extends>QWidget</extends>
   <header>graphpainter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DistanceMatrix</class>
   <extends>QWidget</extends>
   <header>distancematrix.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
